commit a525df06b279539a708aa125c6f71b29cba35f6a
Author: Kelly McLaughlin <kelly.mclaughlin@joyent.com>
Date:   2019-05-14T07:30:21-06:00 (5 months ago)
    
    MANTA-4272 Boray does not pass postgres IP and port info to cueball

diff --git a/src/main.rs b/src/main.rs
index cb77b26..443772f 100644
--- a/src/main.rs
+++ b/src/main.rs
@@ -68,8 +68,8 @@ fn main() {
     let pg_config = PostgresConnectionConfig {
         user: Some(user.into()),
         password: None,
-        host: None,
-        port: None,
+        host: Some(pg_ip.to_string()),
+        port: Some(pg_port.into()),
         database: Some(pg_db.into()),
         application_name: Some(application_name.into())
     };
